docs/Installation.md

```diff
@@ -15,19 +15,23 @@ The macOS `.pkg` installer also installs Homebrew to its default prefix (`/opt/h
   [Xcode](https://itunes.apple.com/us/app/xcode/id497799835) <sup>[3](#3)</sup>
 * The Bourne-again shell for installation (i.e. `bash`) <sup>[4](#4)</sup>
 
-## Git Remote Mirroring
+## Advanced Configuration
 
-You can use geolocalized Git mirrors to speed up Homebrew's installation and `brew update` by setting `HOMEBREW_BREW_GIT_REMOTE` and/or `HOMEBREW_CORE_GIT_REMOTE` in your shell environment with this script:
+The Homebrew installer offers various configuration settings. **Most users can skip this section and instead follow the instructions on the [homepage](https://brew.sh)!** If you don't know what these settings mean then you likely don't need to set them as the defaults work great for nearly all users.
```

Comment:
```suggestion
The Homebrew installer offers various configuration settings. **Most users can skip this section and instead follow the instructions on the [homepage](https://brew.sh)!**
```

I feel like this line probably isn't necessary since the previous line says that this can be skipped for most users and the next section explicitly proves the git mirrors as solutions to Github.com connection issues.